edx/edx-ora2
# feat: jquery-ui confirm

**TL;DR -** Replace window.confirm calls with a custom jquery-ui confirm

JIRA: [AU-78](https://openedx.atlassian.net/browse/AU-78)

**What changed?**

- Replace window.confirm calls with a custom jquery-ui confirm 

**Testing Instructions**

Three confirms were replaced:
- Deleting a file
- Submitting a response
- Grading a team asessment

- Navigate to an individual ORA with files enabled
- Upload some files. Delete some and verify that the new modal confirm appears.
- Submit your response. Verify that the modal appears
- Navgate to a team ORA
- Submit a response
- Click "Grade available responses" and grade the response. Verify that the modal appears when you attempt to submit the grade.

**Screenshots**
- submitting a submission
![image](https://user-images.githubusercontent.com/1639231/138964864-e2868c72-ab2d-438e-ad94-62ab10982b8b.png)
- deleting a file
![image](https://user-images.githubusercontent.com/1639231/138964885-06eb0000-7a87-4f0a-a447-1f31faa6a4b4.png)
- grading a team submission
![image](https://user-images.githubusercontent.com/1639231/138964911-e4b32123-0119-44e7-9f9f-d5a08999fd66.png)

**Reviewer Checklist**

Collectively, these should be completed by reviewers of this PR:

- [x] I've tested the new functionality on:

FYI: @edx/masters-devs-gta
